index.js

```diff
@@ -2,19 +2,31 @@ import { on, html, removeClass, addClass } from 'domassist';
 import Domodule from 'domodule';
 
 export default class PrettyErrors extends Domodule {
-  getRules() {
-    return null;
-  }
-
   get required() {
     return {
       options: ['event']
     };
   }
 
+  getRules() {
+    return null;
+  }
+
+  getErrors() {
+    return null;
+  }
+
   postInit() {
-    on(document.body, this.options.event, this.onError.bind(this));
-    on(document.body, `${this.options.event}:hide`, this.onHide.bind(this));
+    const errors = this.getErrors();
+
+    if (errors) {
```

Comment:
This if then becomes redundant since it will always be an array with at least one value.